lib/v2/screens/dashboard/wallet/interactor/viewmodels/wallet_bloc.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:bloc/bloc.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/dashboard/wallet/interactor/mappers/user_account_state_mapper.dart';
+import 'package:seeds/v2/screens/dashboard/wallet/interactor/usecases/get_user_account.dart';
+import 'package:seeds/v2/screens/dashboard/wallet/interactor/viewmodels/wallet_event.dart';
+import 'package:seeds/v2/screens/dashboard/wallet/interactor/viewmodels/wallet_state.dart';
+
+/// --- BLOC
+class WalletBloc extends Bloc<WalletEvent, WalletState> {
+  WalletBloc() : super(WalletState.initial());
+
+  @override
+  Stream<WalletState> mapEventToState(WalletEvent event) async* {
+    if (event is OnLoadWalletData) {
+      // yield state.copyWith(pageState: PageState.loading);
```

Comment:
commented code?

Reply:
Thats Nik code, I think he have it there as a holder.

Reply:
Yes Bro but these lines were write just to simulate the current behaivor already has what you did haha. You can remove those lines now, in Fact Nick puts just the loadig one I add the succes to change the state.

Reply:
commented code was remove